### PR TITLE
Fe searching for chefs

### PR DESCRIPTION
**Pull request by giftofgrub**
Test out backend for searching chefs.

Steps to test:
- go to /chefs, you should see all chefs
- apply filter on cuisine. chef has stub cuisines of ""Japanese, Chinese, Mexican"". So, choosing those that do not apply should return with no chefs.
- apply distance filter by typing in Toronto, CA. Chef has stub location around the area of toronto.
- Choosing a filter of 1 should return no chefs. But if slider is put to 0. The filter should not apply, and will return all chefs

### CODE REVIEW

File: client/src/components/ChefFilters.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+      setUserAddress("");
+    }
+  };
+
+  const addSelectedCuisine = (cuisine) => {
+    setSelectedCuisines([...selectedCuisines, cuisine]);
+  };
+
+  const removeSelectedCuisine = (cuisine) => {
+    let filteredCuisines = selectedCuisines.filter((c) => c !== cuisine);
```
Comment (skassam21) [inline client/src/components/ChefFilters.js:50]: Just a small think, let's use const whenever you can (favor const vs let)

File: client/src/components/ChefSearch.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  const [userCoordinates, setUserCoordinates] = useState(null);
+  const [selectedCuisines, setSelectedCuisines] = useState([]);
+  const [distanceFilter, setDistanceFilter] = useState(null);
+
+  const classes = useStyles();
+  useEffect(() => {
+    // do api call
+    searchChefs();
+    // to change later
+    setDummyData();
```
Comment (skassam21) [inline client/src/components/ChefSearch.js:60]: Are you planning on removing this dummy data in a future PR?

File: client/src/pages/Chefs.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ChefSearch from "components/ChefSearch";
+import LoginForm from "components/LoginForm";
+import Navbar from "components/Navbar"
+
+const Chefs = ({ props }) => {
+  return (
+    <>
+      <Navbar/>
```
Comment (skassam21) [inline client/src/pages/Chefs.js:9]: Does the navbar need to be added in each component? This component should be in LoggedInComponent, so it shouldn't need to be here right?

Comment (giftofgrub) [reply client/src/pages/Chefs.js:9]: Yup, this needs to be taken out. I had that in there assuming that the user could search for chefs without being logged in.

File: client/src/components/ChefSearch.js (hunk 2 above)
Comment (giftofgrub) [reply client/src/components/ChefSearch.js:60]: Yup, just had to put some stub data for cuisines

**Pull request merged**